packages/quantic/force-app/main/default/lwc/quanticResultPrintableUri/quanticResultPrintableUri.html

```diff
@@ -7,7 +7,7 @@
       <template for:each={foldedParents} for:item="parent">
         <div class="result-parenturi__container slds-grid slds-grid_vertical-align-center" key={parent.id}>
           <template if:true={parent.isFolded}>
-            <a onclick={expandParents}>...</a>
+            <button class="slds-button expand-parenturi__container" variant="base" label="..." title="Expand URI parts" aria-label="Expand URI parts" onclick={expandParents}>...</button>
```

Comment:
@acarpentier01 what do you think of the text "Expand URI parts"?

Reply:
I think "Expand URI" or "Expand path" would do the trick @drosenfeld-coveo. @jpmarceau Do you agree?

Reply:
I think so, unless by `part` you mean something technical, like a CSS part?

Reply:
I dont think the "part" has a specific meaning, I like "Expand URI"

Reply:
@erocheleau Just to confirm, is this is a URI or a breadcrumb? If it's a URI, then please go with "Expand URI" @drosenfeld-coveo.